index.js

```diff
@@ -11,6 +11,11 @@ function md5(str) {
 }
 
 function relPath(base, filePath) {
+	if ( path.sep === '\\') {
+		base = base.replace(/\\/gi, '/');
+		filePath = filePath.replace(/\\/gi, '/');
```

Comment:
If you're replacing all the slashes here, you should remove them from below (there are at least a couple other places that do this same replacement).